pyproject.toml

```diff
@@ -1,15 +1,180 @@
 [build-system]
-requires = [
-    "setuptools >= 45",
-    "setuptools_scm[toml] >= 6.2",
-    "wheel"
-]
-build-backend = "setuptools.build_meta"
-
-[tool.setuptools_scm]
-write_to = "src/eddymotion/_version.py"
-write_to_template = """\
-\"\"\"Version file, automatically generated by setuptools_scm.\"\"\"
-__version__ = "{version}"
-"""
-fallback_version = "0.0"
+requires = ["hatchling", "hatch-vcs", "nipreps-versions"]
+build-backend = "hatchling.build"
+
+
+[project]
+name = "eddymotion"
+description = "Pure python eddy-current and head-motion correction for dMRI, an extension of QSIprep's SHOREline algorithm (Cieslak, 2020) to multiple diffusion models."
+readme = "README.rst"
+authors = [{name = "The NiPreps Developers", email = "nipreps@gmail.com"}]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering :: Image Recognition",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+license = "Apache-2.0"
+requires-python = ">=3.8"
+dependencies = [
+    "dipy>=1.3.0",
+    "joblib",
+    "nipype>= 1.5.1, < 2.0",
+    "nitransforms>=21.0.0",
+    "nireports",
+    "numpy>=1.17.3",
+    "nest-asyncio>=1.5.1",
+    "scikit-image>=0.14.2",
+    "scipy>=1.8.0",
+]
+dynamic = ["version"]
+
+[project.urls]
+Documentation = "https://www.nipreps.org/eddymotion"
+Home = "https://github.com/nipreps/eddymotion"
+NiPreps = "https://www.nipreps.org/"
+
+[project.optional-dependencies]
+doc = [
+    "nbsphinx",
+    "packaging",
+    "pydot >= 1.2.3",
+    "pydotplus",
+    "sphinx >= 2.1.2",
+    "sphinx-argparse",
+    "sphinx_rtd_theme",
+    "sphinxcontrib-apidoc ~= 0.3.0",
+    "sphinxcontrib-napoleon",
+    "sphinxcontrib-versioning"
```

Comment:
`furo` needs to be added here: https://github.com/nipreps/eddymotion/blob/4b632ea345faa69088c03447dc9e707d93c62c32/docs/conf.py#L115